File: moban_java/src/main/java/org/jdk/project/dto/config/AreaUpsertRequest.java
package org.jdk.project.dto.config;

import java.util.List;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AreaUpsertRequest {
  private Long loungeId;
  private String areaName;
  private String coordinate;
  private Integer maxCapacity;
  private String isGuide;
  private String isShow;
  private Integer enable;
  private String remark;
  private List<AreaDetailUpsertRequest> configAreaDetailList;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/AreaUpsertRequest.kt
package org.jdk.project.dto.config

data class AreaUpsertRequest(
    var siteId: Long? = null,
    var areaName: String? = null,
    var coordinate: String? = null,
    var maxCapacity: Int? = null,
    var isShow: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
